src/plot.js

```diff
@@ -258,8 +266,11 @@ export function plot(options = {}) {
           let index = null;
           if (indexes) {
             index = indexes[facetStateByMark.has(mark) ? f.i : 0];
+            const {fx, fy} = index;
             index = mark.filter(index, channels, values);
             if (index.length === 0) continue;
+            index.fx = fx; // TODO do we need to expose the facet values?
+            index.fy = fy; // TODO do we need to expose the facet values?
```

Comment:
I was confused by this, because the https://observablehq.com/@observablehq/autoplot-matrix example relies on the fx, fy properties being attached to the index (as mentioned in #1263).

The explanation of why it works in that example, is because the custom SubPlot mark does not filter, these properties were left as-is. If calling mark.filter removes them, it seems like a bug, so it's nice that we add them back. Alternatively, we could require mark.filter to be more cautious? (i.e. move this part into  https://github.com/observablehq/plot/blob/fd1837179bb7a19f83e306c0a293b2caaf9154a2/src/mark.js#L86)

Reply:
I think it’s okay to always have these properties available. I don’t see any reason to hide them, at least, and it’s good to be consistent.

Reply:
Oh I definitely want them—what I meant is that I thought they were already there (always), when in fact they could be removed if the mark was filtering the index.